sites/en/installfest/editors.step

```diff
@@ -1,12 +1,12 @@
 message <<-MARKDOWN
 
-There are a number of different editors designed for programming. You may already have a favorite editor; if so you can continue using that editor. If you have never used a programming editor, there are a number of good editors available, several of them free of charge. Most Railsbridge instructors use the Sublime editor, so you may want to install Sublime; if you end up asking for help during your class it may be easier for an instructor or TA to help you if you are using Sublime. Here is a list of some of the more widely used editors:
+There are a number of different editors designed for programming. You may already have a favorite editor; if so you can continue using that editor. If you have never used a programming editor, there are a number of good editors available, several of them free of charge. Most RailsBridge instructors use the Atom editor, so you may want to install Atom; if you end up asking for help during your class it may be easier for an instructor or TA to help you if you are using Atom. Here is a list of some of the more widely used editors:
```

Comment:
Since we're making some text changes, it might make sense to replace "Most RailsBridge instructors" with "Many RailsBridge volunteers", since I don't think we've done a poll of what editors every volunteer uses.

:thinking: We might also want to remove some of the older alternate suggestions below, but that might be more ✨ controversial ✨  of a project than we should take on right now.